apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-docs.component.html

```diff
@@ -1,44 +1,48 @@
-<fd-docs-section-title [id]="'def'" [componentName]="'Radio Button Group'">
-    Radio Button Group with Radio buttons as Content
+<fd-docs-section-title [id]="'projection'" [componentName]="'Radio Button Group'">
+    Radio Button Group With Radio Buttons Passed as Content.
 </fd-docs-section-title>
 
-<description>This example shows Radio Button Group implementaion with Content as Radio Buttons</description>
+<description>
+    Platform Radio Buttons Passed as Content
+</description>
 <component-example>
     <fdp-platform-radio-group-content-example></fdp-platform-radio-group-content-example>
 </component-example>
 <code-example [exampleFiles]="contentRadioGroup"></code-example>
 
-<fd-docs-section-title [id]="'def'" [componentName]="'Disabled Radio Button Group'">
-    Examples for Radio Button Group with List of values
+<fd-docs-section-title [id]="'list'" [componentName]="'Disabled Radio Button Group'">
+    Radio Button Group Creates Radio Buttons With List of String Values.
 </fd-docs-section-title>
 <description>
-    This example shows Regular Radio Button Group implementaion with list of string Values.<br />
+    Creating Platform Radio Buttons with List of String Values.<br />
     eg. ['Summer', 'Winter', 'Autumn'], Radio buttons are created with these values.
 </description>
 <component-example>
     <fdp-platform-radio-group-list-example></fdp-platform-radio-group-list-example>
 </component-example>
 <code-example [exampleFiles]="listRadioGroup"></code-example>
 
-<fd-docs-section-title [id]="'def'" [componentName]="'Radio Button Group'">
-    Examples for Radio Button Group with List of objects (SelectItem Object)
+<fd-docs-section-title [id]="'listobject'" [componentName]="'Radio Button Group'">
+    Radio Button Group Creates Platform Radio Buttons With List of Objects. <br />
 </fd-docs-section-title>
 
 <description>
-    This example shows Regular Radio Button Group implementaion with list of Objects of SelectItem.<br />
-    Radio buttons are created with these values.
+    Creating Platform Radio Buttons with List of Object.<br />
+    Here value, label and disabled properties of SelectItem object or LookupKey and DisaplayKey of any objects are used
+    to Create Individual Radio Buttons.
 </description>
 <component-example>
     <fdp-platform-radio-group-list-items-example></fdp-platform-radio-group-list-items-example>
 </component-example>
 <code-example [exampleFiles]="listItemsRadioGroup"></code-example>
 
-<fd-docs-section-title [id]="'def'" [componentName]="'Disabled Radio Button Group'">
-    Examples for Disable Radio Button Group
+<fd-docs-section-title [id]="'disabled'" [componentName]="'Disabled Radio Button Group'">
+    Radio Button Group for Disable Radio Buttons. Here One or More Radio Buttons Can be Disabled and Others Can be
```

Comment:
Again, "Here" is awkward in this sentence. You can actually remove it entirely, and it's also not necessary to capitalize every word except component titles (headers). "This radio button group displays disabled radio buttons. One or more radio buttons can either be in a disabled or enabled state."

Reply:
corrected.